Octokit/Models/Response/TimelineEventInfo.cs

```diff
@@ -40,9 +41,25 @@ public TimelineEventInfo(long id, string nodeId, string url, User actor, string
         public Label Label { get; protected set; }
         public User Assignee { get; protected set; }
         public Milestone Milestone { get; protected set; }
+
+        /// <summary>
```

Comment:
:art: this is great - thanks for documenting both these current parts as well as the new property!